new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+     filename and the position in the file.  Remember the line number for Step
+     3.
+
+NOTE:  You may see the cursor position in the lower right corner of the screen
+       This happens when the ['ruler']('ruler') option is set (see  :help 'ruler'  )
+
+  2. Press [«G»](G)  to move you to the bottom of the file.
+     Type  [«gg»](gg)  to move you to the start of the file.
+
+  3. Type the number of the line you were on and then  «G» .  This will
+     return you to the line you were on when you first pressed <Ctrl-g>.
+
+  4. If you feel confident to do this, execute steps 1 through 3.
+
+# Lesson 4.3: MATCHING PARENTHESES SEARCH
+
+** Type  %  to find a matching ),], or } . **
+
+  1. Place the cursor on any (, [, or { in the line below marked --->.
+
+  2. Now type the  [«%»](%)  character.
+
+  3. The cursor will move to the matching parenthesis or bracket.
+
+  4. Type  «%»  to move the cursor to the other matching bracket.
+
+  5. Move the cursor to another (,),[,],{ or } and see what  %  does.
+
+---> This ( is a test line with ('s, ['s ] and {'s } in it. )) {expect:NULL}
+
+NOTE: This is very useful in debugging a program with unmatched parentheses!
+
+
+# Conclusion
+
+So far we have learned how to do basic text operation, we know how to navigate
+the text quickly, how to insert text and finally we learned how to transplant
+text. We also learned that Vim actions are actually sentences constructed
+acctording to a grammar. Learning how to express our thoughts in this Vim
+language is the key to getting really producive, allowing you to execute actions
+as fast as you can think them.
+
+=========  =================        ========  ========================
+ Motion          Effect             Operator  Effect
+=========  =================        ========  ========================
+    h      left                        x      Delete current character
+    j      down                        i      Insert before cursor
+    k      up                          a      Append after cursor
+    l      right                       o      Open a new line
+    w      next word                   d      Delete text with motion
+    e      end of word                 y      Yank text with motion
+    b      beginning of word           c      Change text with motion
+    0      beginning of line           r      Replace character/text
+    $      end of line                 u      Undo action
+l <SPACE>  current character         <C-R>    Redo action
+    _      current line                p      Put yanked/deleted text
+    f      fing character           ========  ========================
+    t      'till character
+    i      inside something         Pressing an operator twice acts as
+    a      around something         a motion on the entire line.
+   gg      first line
+    G      last line
+    %      matching braces
+=========  =================
+
+
+
+# Part 2: EX COMMANDS
+
+** Command mode is where we can execute powerful commands **
```

Comment:
Small reminder about this comment.
I think it sounds fine (but plain) if you remove the 
## powerful.